### DISCUSSION TITLE
SSD Transferlearning

### DISCUSSION THREAD

**Discussion by FreddieNotFish** [deepjavalibrary/djl#1797] [Q&A]
Hello, I am struggling a bit with transferlearning SSD-models. I am able to transferlearn Resnet-models etc. But I can't really seem  to figure out how to approach it on SSD models in DJL with my own dataset (I don't know if there is an example for this? (I've seen the resnet example)). I want to use it for boat detection and I think that transferlearning would optimize my system. Thanks!

**Comment by zachgk** [comment]
I don't think we have any other transfer learning examples. Can you give a bit more details about where exactly you are having problems?
